### ISSUE TITLE
Fails to build on Java 9: sun.misc.URLClassPath is gone

### ISSUE THREAD

**Issue by FauxFaux** [OpenHFT/Java-Thread-Affinity#37]
In Java 9, due out in ~two weeks, the `sun.misc.URLClassPath` no longer exists.
It has always been an implementation detail they were free to remove at any time. Please stop using it.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.2:compile (default-compile) on project affinity: Compilation failure: Compilation failure: 
[ERROR] /home/faux/code/Java-Thread-Affinity/affinity/src/main/java/net/openhft/affinity/BootClassPath.java:[22,16] cannot find symbol
[ERROR]   symbol:   class URLClassPath
[ERROR]   location: package sun.misc
[ERROR] /home/faux/code/Java-Thread-Affinity/affinity/src/main/java/net/openhft/affinity/BootClassPath.java:[31,19] cannot find symbol
[ERROR]   symbol:   class URLClassPath
[ERROR]   location: class net.openhft.affinity.BootClassPath
[ERROR] /home/faux/code/Java-Thread-Affinity/affinity/src/main/java/net/openhft/affinity/BootClassPath.java:[31,52] cannot find symbol
[ERROR]   symbol:   class URLClassPath
[ERROR]   location: class net.openhft.affinity.BootClassPath
```

**Comment by FauxFaux** [comment]
I've just noticed https://github.com/OpenHFT/Chronicle-Bytes/issues/40 so maybe someone is already working on it. FYI `chronicle-core` and `chronicle-wire` also fail for me, but I won't be raising bugs.

**Comment by TerryW9** [comment]
Hi FauxFaux,

Thank you for pointing this issue out to us.

We will look into it as soon as possible.

Best Regards,
Terry

**Comment by epickrram** [comment]
Thanks for the report @FauxFaux . This is fixed in da19214.
Java9 compatibility for all OpenHFT projects is currently work-in-progress.
